Dockerfile

```diff
@@ -1,13 +1,28 @@
-# Image: Ubuntu 20.04 Stable, Official Image from Canonical 
+# Image: Ubuntu 20.04 Stable, Official Image from Canonical
 FROM public.ecr.aws/lts/ubuntu:20.04_stable
 
 # Performs updates and installs git, make, curl, python3.8, python3-pip, python3.8-dev and pylint packages
+# Line 13 is required by the spacepy Python package
+# Line >=14 installs cdflib
 RUN apt-get update && \
     apt-get -y upgrade && \
+    apt-get -y install --no-install-recommends -y python3.8 python3-pip python3.8-dev pylint && \
     apt-get -y install git && \
     apt-get -y install make && \
     apt-get -y install curl && \
-    apt-get -y install --no-install-recommends -y python3.8 python3-pip python3.8-dev pylint
+    apt-get -y install wget && \
+    apt-get -y install gfortran && \
+    wget https://spdf.gsfc.nasa.gov/pub/software/cdf/dist/cdf38_0/cdf38_0-dist-cdf.tar.gz && \
+    tar zxvpf cdf38_0-dist-cdf.tar.gz && rm cdf38_0-dist-cdf.tar.gz && \
+    apt-get -y install libncurses5-dev && \
+    apt-get -y install gcc && \
+    cd cdf38_0-dist && \
+    make OS=linux ENV=gnu all && \
+    make INSTALLDIR=/usr/local/cdf install && \
+    cd .. && \
```

Comment:
Next time feel free to make the suggestion as a suggested code change using GitHub. For something small like this it is easier to just suggest the change directly rather than describe it and it saves a back and forth. I can just click accept. See this [article](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes) if you are not familiar. Just click the +/- button in the comment.